WEB-INF/sql/create_tables.sql

```diff
@@ -22,4 +22,15 @@ CREATE TABLE Issues (
   locked      BIT          NOT NULL DEFAULT 0
 );
 
+CREATE TABLE Articles (
+  id          CHAR(36)     PRIMARY KEY,
+  title       VARCHAR(128) NOT NULL,
+  body        TEXT         NOT NULL,
+  answer      TEXT         NOT NULL,
+  helpfulness INT          NOT NULL DEFAULT 0,
+  published   DATE         NOT NULL,
+  updated     DATE         NOT NULL
```

Comment:
For right now, no dates. I'll add them later once I make models automatically update these values.